content/src/content/docs/learn/tutorials/first-endpoint.mdx

```diff
@@ -0,0 +1,355 @@
+---
+type: Tutorial
+title: Our first Effect endpoint
+tags:
+  - some
+  - tags
+sidebar:
+  order: 2
+---
+
+import { Aside } from "@astrojs/starlight/components"
+
+The first step to writing Effect is to modify our Express app to allow it to
+serve both Effect and Express endpoints at the same time. We need to first add
+some dependencies to our project:
+
+```shell showLineNumbers=false
+bun add effect @effect/platform @effect/platform-node
+```
+
+<Aside type="note" title="Why @effect/platform-node?">
+The `@effect/platform-node` there isn't a mistake. Provided your runtime
+supports `node:http`, you can use `@effect/platform-node` to run your Effect
+app.
+
+Effect does have an `@effect/platform-bun`, which uses Bun's own HTTP server,
+but to make sure this tutorial is as widely applicable as possible, I'm going
+to use `@effect/platform-node`.
+</Aside>
+
+Then make the highlighted changes to our Express app:
+
+```ts twoslash ins={3-5,16-26} del={13-15} title=index.ts showLineNumbers=false
+import express from "express"
+
+import { NodeHttpServer, NodeRuntime } from "@effect/platform-node"
+import { HttpRouter } from "@effect/platform"
+import { Effect } from "effect"
+
+const app = express()
+
+app.get("/health", (req, res) => {
+  res.type("text/plain").send("ok")
+})
+
+app.listen(3000, () => {
+  console.log("Server is running on http://localhost:3000")
+})
+const router = HttpRouter.empty
+
+const main = Effect.gen(function* () {
+  app.use(yield* NodeHttpServer.makeHandler(router))
+  app.listen(3000, () => {
+    console.log("Server is running on http://localhost:3000")
+  })
+  yield* Effect.never
+})
+
+NodeRuntime.runMain(main)
+```
+
+And we can run the resulting code like so:
+
+```shell showLineNumbers=false
+bun index.ts
+```
+
+Our Express `/health` handler still works as expected, but we've wrapped the
+Express app in a way that will allow us to define endpoints using Effect
+while still responding to our existing endpoints in Express.
+
+There's a lot of new things happening at the bottom of our file. What is
+`Effect.gen`, what are those `yield*`s doing, what is `Effect.never`. For the
+time being we're going to set these questions aside. It looks intimidating now,
+but if you stick with this tutorial to the end, it will not only make sense, but
+start to become second nature.
+
+## Our new `/health` handler
+
+The Effect equivalent of our Express `/health` handler looks like this:
+
+```ts twoslash showLineNumbers=false
+import { Effect } from "effect"
+// ---cut---
+import { HttpServerResponse } from "@effect/platform"
+
+const health = Effect.sync(() => {
+  return HttpServerResponse.text("ok")
+})
```

Comment:
```suggestion
const health = HttpServerResponse.text("ok")
```

I still think we should get rid of the `Effect.sync` wrapper here. It's superfluous since the code works without it, and adds one more thing you have to explain.